includes/amp-helper-functions.php

```diff
@@ -414,23 +414,60 @@ function amp_is_available() {
 	}
 
 	$warn = function () {
-		static $warned = false;
-		if ( $warned ) {
+		static $already_warned_sources = [];
+
+		// Gather the themes and plugins responsible for calling the function incorrectly.
+		$closest_source = [
+			'type' => '',
+			'name' => '',
+		];
+
+		foreach ( debug_backtrace( 0 ) as $call_stack ) { // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_debug_backtrace
```

Comment:
@westonruter You're passing `0` here to the `$options` argument of `debug_backtrace()`. Just to be clear, I assume this was meant to lift the second argument `$limit` ?

Reply:
...or was the goal to unset the default `DEBUG_BACKTRACE_PROVIDE_OBJECT` option to save processing time? I think that would actually be the wanted behavior here as well.
Using `0` here is very ambiguous, though, and will warrant at least a clarifying comment.

I'll adapt as needed.